docs/website/docs/reference/mlir-dialects/index.md

```diff
@@ -19,11 +19,14 @@ these dialects can be found in the
 Dialect                     | Description
 --------------------------- | -----------
 [Check](./Check.md)         | Defines assertions for IREE tests
+[Encoding](./Encoding.md)   | Tensor encoding attributes and related ops
 [Flow](./Flow.md)           | Models execution data flow and partitioning
 [HAL](./HAL.md)             | Represents operations against the IREE HAL[^1]
 [HAL/Inline](./HALInline.md) | Inline HAL interop runtime module dialect
 [HAL/Loader](./HALLoader.md) | HAL inline executable loader runtime module dialect
 [IO/Parameters](./IOParameters.md) | External parameter resource management APIs
+[IREECodegen](./IREECodegen.md) | Attributes used by IREE code generation
+[IREEGPU](./IREEGPU.md) | Attributes used by GPU focused IREE code generation
```

Comment:
I don't think it's just attributes. It's some operations as well

Reply:
IREECodegen also has an op for ukernels. Something like `Common functionality used by [GPU focused] IREE code generation`.

Reply:
Thanks, updated the summaries.